packages/in_app_purchase/lib/src/in_app_purchase_tizen_platform_addition.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:in_app_purchase_platform_interface/in_app_purchase_platform_interface.dart';
+
+import '../billing_manager_wrappers.dart';
+
+/// Contains InApp Purchase features that are only available on SamsungCheckout.
+class InAppPurchaseTizenPlatformAddition extends InAppPurchasePlatformAddition {
+  /// Creates a [InAppPurchaseTizenPlatformAddition] which uses the supplied
+  /// `BillingManager` to provide Tizen specific features.
+  InAppPurchaseTizenPlatformAddition(this._billingManager);
+
+  final BillingManager _billingManager;
+
+  /// Set all request parameters that SamsungCheckout DPI Portal needed.
+  ///
+  /// The `appId` is your application id, must be required.
+  ///
+  /// The `serverType` is Billing server type, must be required.
+  ///
+  /// The `countryCode` is your device country code.
+  /// Use it when call `queryProductDetails`, `restorePurchases`, `buyConsumable` and `buyNonConsumable`.
+  ///
+  /// The `pageSize` is the number of products retrieved per page, more than 1 and less than 100.
+  /// Use it when call `queryProductDetails`.
+  ///
+  /// The `pageNum` is the requested page number, more than 1.
```

Comment:
"more than 1" means a number larger than 1, that is, 2 or higher.

How about `(>= 1)`, `(1-N)`, or `(min 1)`?